packages/client/src/pages/login.tsx

```diff
@@ -0,0 +1,12 @@
+import { Box } from '@mui/material';
+import { useEffect } from 'react';
+
+export const Login = () => {
+  const loginUrl = `${import.meta.env.VITE_AUTHSERVICE_URL}`;
+
+  useEffect(() => {
+    window.location.href = loginUrl;
+  });
```

Comment:
This useEffect needs to be scoped, otherwise it could repeat

```suggestion
  useEffect(() => {
    window.location.href = loginUrl;
  }, []);
```